src/main/java/com/solvd/webtesting/page/SuccessRegistrationPage.java

```diff
@@ -0,0 +1,19 @@
+package com.solvd.webtesting.page;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class SuccessRegistrationPage extends AbstractPage {
+    @FindBy(css = ".styles_successTitle__1IDi2")
+    private ExtendedWebElement successRegistration;
+
+    public SuccessRegistrationPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public ExtendedWebElement getSuccessRegistration() {
+        return successRegistration;
+    }
```

Comment:
The same: Please don't return ExtendedWebElement, if use need to do some manipulations with elements, u need to do this on page where it present.

Reply:
resolved for all unused getElements